src/Tests/Microsoft.DotNet.GenAPI.Tests/CSharpFileBuilderTests.cs

```diff
@@ -131,6 +131,7 @@ record struct RecordStruct { }
                     public readonly ref struct PublicReadonlyRefStruct { }
                 }
                 """,
+                // It's expected that record structs are expanded - they're effectively syntactic sugar that is not persisted in metadata
```

Comment:
Do you mean the extra newline between the brackets?

Reply:
I mean all the compiler generated members listed for these types.